common/v2/components/NetworkSelectDropdown.tsx

```diff
@@ -11,21 +11,22 @@ import { Typography, Dropdown } from 'v2/components';
 interface Props {
   network: string | undefined;
   accountType?: WalletId;
+  className?: string;
   onChange(network: NetworkId): void;
 }
 
 const DropdownContainer = styled('div')`
   .is-open > .Select-control > .Select-multi-value-wrapper > .Select-input:only-child {
     transform: translateY(0%);
-    padding: 16px 15px 16px 15px;
+    padding: 12px 12px;
     position: inherit;
   }
 `;
 
 const SContainer = styled('div')`
   display: flex;
   flex-direction: row;
-  padding: 16px 15px 16px 15px;
+  padding: 12px 12px;
```

Comment:
1. Since they are identical prefer shorthand: `padding: 12px;`
2. Do we have a theme/SPACING that is equivalent?

Reply:
We have `FONT_SIZE.XS` but not sure how much sense that makes 😄